### DISCUSSION TITLE
Cannot start Hasura in the community/boilerplates/observability/enterprise demo due to a config error

### DISCUSSION THREAD

**Discussion by deinspanjer** [hasura/graphql-engine#10290] [Question]
After creating a .env file and putting in our HASURA_GRAPHQL_EE_LICENSE_KEY as described in the readme, I try to run `docker-compose up -d` but get the following error in the console output of hge:
`level=fatal msg="error applying metadata \n{\n  \"error\": \"OpenTelemetry status must be either \\\"enabled\\\" or \\\"disabled\\\"\",\n  \"path\": \"$.args.metadata.opentelemetry.status\",\n  \"code\": \"parse-failed\"\n}`

Has this been tested recently to see if it still works?

**Comment by hgiasac** [comment]
Hi @deinspanjer
Sorry about that. The metadata config of the observability boilerplate is for v2.39
Can you change the `HGE_IMAGE` version in [.env](https://github.com/hasura/graphql-engine/blob/0f245de56673c7251a809f51b51836d8ef90ed7e/community/boilerplates/observability/enterprise/dotenv#L2) file and try again?
